lib/modules.nix

```diff
@@ -358,12 +358,30 @@ let
         options = checked options;
         config = checked (removeAttrs config [ "_module" ]);
         _module = checked (config._module);
+        graph =
+          let
+            collected =
+              collectModules class (specialArgs.modulesPath or "") (regularModules ++ [ internalModule ])
+                (
+                  {
+                    inherit
+                      lib
+                      options
+                      config
+                      specialArgs
+                      ;
+                    _class = class;
+                  }
+                  // specialArgs
+                );
+          in
+          collected.graph;
```

Comment:
This kinda sucks because the expression of this binding already exists in this file.

Reply:
Done.